### DISCUSSION TITLE
Good practices as a freelancer

### DISCUSSION THREAD

**Discussion by CornerAppGuy** [supabase/supabase#22516] [Questions] [answered]
Hello there,
Just beginning as a freelancer, I'm currently developping an app for a client with Supabase and Weweb. My question is how to deal with client access to Supabase ? Is it possible to give access or share in the Starter plan ? Should I create the DB directly within the client's supabase account ? 
Thanks in advance

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/platform/access-control

**Reply by CornerAppGuy** [reply]
Thankss
